webapp/package.json

```
@@ -0,0 +1,18 @@
+{
+  "devDependencies": {
+    "@parcel/packager-raw-url": "2.12.0",
+    "@parcel/transformer-webmanifest": "2.12.0",
+    "tailwindcss": "^3.4.10"
```

Use fixed version, you can prevent incompatibility issues in the future
```suggestion
    "tailwindcss": "3.4.10"
```